spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -1739,6 +1749,20 @@ private void sleepFor(Duration duration) {
 			}
 		}
 
+		private void enforceRebalanceIfNecessary() {
+			try {
+				if (KafkaMessageListenerContainer.this.enforceRebalanceRequested.get()) {
```

Comment:
Therefore then here: `KafkaMessageListenerContainer.this.thisOrParentContainer.enforceRebalanceRequested.get()`